packages/documentation/src/stories/components/language-switch/language-switch.stories.ts

```diff
@@ -26,6 +27,12 @@ const meta: MetaComponent<HTMLPostLanguageSwitchElement> = {
         category: 'content',
       },
     },
+    type: {
+      description: 'View type of the language switch.',
+      table: {
+        category: 'content',
+      },
+    },
```

Comment:
This override is unnecessary, this information is generated from the tsdocs in the components package.
```suggestion
```